smoke/IotEdgeQuickstart/details/TwinTestConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace IotEdgeQuickstart.Details
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class TwinTestConfiguration
+    {
+        [JsonProperty(PropertyName = "moduleId")]
+        public string ModuleId { get; set; }
+
+        [JsonProperty(PropertyName = "twinTest")]
+        public Dictionary<string, Dictionary<string, string>> TwinTest;
```

Comment:
Suggest making it 

TwinCollection Desired {get;}
TwinCollection ExpectedReported {get;}

Reply:
Done! Not with TwinCollection, but with TwinProperties.